### PR TITLE
Labs/update-sonarcloud

### PR DESCRIPTION
**Pull request by kevin-hinz**
This PR includes one commit updating the tutorial through Exercise 1.

The second commit manages the images: adding the new/current screenshots and deleting all old images, including those added by other authors in previous commits.

### CODE REVIEW

**Comment by kevin-hinz** [comment]
@sriramdasbalaji, This branch was prepared by the Sonar Documentation team to correct, clean, update, and improve the whole tutorial.

Can you please have a look and help us merge this PR?

**Comment by surajshenoy** [comment]
@antonio-garcia-sonarsource @kevin-hinz . We are looking into the changes and will merge once the review is done

File: labs/vstsextend/sonarcloud/readme.md [hunk 1]
```diff
@@ -10,400 +10,425 @@ folder: /labs/vstsextend/sonarcloud/
...
+      ![Create pull request](images/3/azure-create-pr.png)
 
-      If the pull request integration is correctly configured the UI will show that an analysis build is in progress.
+      If the pull request integration is configured correctly, the UI will show that an analysis build is in progress.
 
-     ![Analysis in progress](images/azdo_pr_analysis_in_progress.png)
+     ![Analysis in progress](images/3/azure-pr-analysis-in-progress.png)
 
-5. Review the results of the Pull Request analysis
+[* SECTION IMAGES MUST BE UPDATED BECAUSE THE NEXT SECTION, 3, IS ALREADY IMPLIMENTED AND IS THUS, FAILING THE PR BECAUSE THE QUALITY GATE FAILED... AND WE SHOULD BE ABLE TO PASS THE PR IN THIS SECTION]
```
Comment (surajshenoy) [inline labs/vstsextend/sonarcloud/readme.md:403]: Please remove or modify this comment. It seems that it is a note for the author!

File: labs/vstsextend/sonarcloud/readme.md [hunk 2]
```diff
@@ -418,60 +443,56 @@ Prerequisites :
...
 
-   ![pre_deploy_conditions_settings](images/ex4/predeploy_conditions_settings.PNG)
+1. Select **Enabled** beside **Gates**
+1. Select **+ Add**, then choose the **SonarCloud Quality Gate status check** Deployment gate. 
 
-1. In order to have the fastest result possible for this exercise, we recommend you to set the evaluations options as per this screenshot (See [how Gate evaluation flow works](https://docs.microsoft.com/en-us/azure/devops/pipelines/release/approvals/gates?view=azure-devops), as per Microsoft Documentation)
+   ![define predeployment conditions](images/4/azure-pre-deployment-conditions-gates.PNG)
 
-   ![gate_setting](images/ex4/gate_setting.PNG)
+1. [* BETTER DESCRIBE FOR EVALUATION OPTIONS; UPDATE TO MENTION "One successful gates...."] To have the fastest result possible for this exercise, we recommend you to define the Evaluation options as shown in this screenshot. Check the Microsoft Documentation for more information about how [Deployments gates](https://docs.microsoft.com/en-us/azure/devops/pipelines/release/approvals/gates?view=azure-devops) in Azure DevOps.
```
Comment (surajshenoy) [inline labs/vstsextend/sonarcloud/readme.md:463]: Content inside the square brackets looks like it is not necessary. Please remove or modify

**Comment by surajshenoy** [comment]
In [Exercise 4], Images of Step 3 to 13 are not loading. Please have a check
image

**Review by surajshenoy — CHANGES_REQUESTED** [review]

**Comment by kevin-hinz** [comment]
> In [Exercise 4], Images of Step 3 to 13 are not loading. Please have a check image

The error was in the file extension and the images should render now.

**Review by sriramdasbalaji — APPROVED** [review]

**Pull request merged**